ingest/CreateArrayImportTsvs.wdl

```diff
@@ -4,10 +4,11 @@ workflow CreateArrayImportTsvs {
 
   input {
     File input_vcf
-    String? probe_info_table
-    String? probe_info_file
+    File? probe_info_table
+    File? probe_info_file
     String output_directory
     File sampleMap
+    Boolean use_compressed_data = false
```

Comment:
i think the default should be true, since that is how it will be run in production. (fyi, i'm changing the default value in the tool)